scripts/generateDescriptions.js

```diff
@@ -30,11 +37,40 @@ async function loadDoc(url) {
   return code;
 }
 
-async function generateDescriptions() {
-  const code = (await Promise.all(DOC_URL.split(',').map(loadDoc))).join('\n\n');
+function compare(current, fileName) {
+  if (!fs.existsSync(fileName)) {
+    console.error(`file ${fileName} doesn't exit`);
+    process.exit(1);
+  }
+  if (!current) {
+    console.warn(`current text is empty, indicator a downloading error`, fileName);
+    return true;
+  }
+  const stored = fs.readFileSync(fileName).toString();
+  if (stored !== current) {
+    console.error(`file ${fileName} and downloaded version, doesn't match -> forgotten to run "npm run gen"?`);
```

Comment:
Can we change the message to be something like, "File ${fileName} is out of date and needs to be synced.  Open a new PR and commit the output of `npm run gen`", otherwise the people will include the descriptions in whichever PR they're working on.